rotkehlchen/rotkehlchen.py

```diff
@@ -353,6 +355,13 @@ def unlock_user(
             connect_at_start=optimism_nodes,
         )
         optimism_manager = OptimismManager(optimism_inquirer)
+        polygon_pos_nodes = self.data.db.get_rpc_nodes(blockchain=SupportedBlockchain.POLYGON_POS, only_active=True)  # noqa: E501
```

Comment:
You need to add a POLYGON_TODO to populate those nodes too. Most probably at the DB upgrade. And we will need to also update our default nodes.